qtfred/src/main.cpp

```diff
@@ -102,7 +102,7 @@ int main(int argc, char* argv[]) {
 	// Expect that the platform library is in the same directory
 	QCoreApplication::addLibraryPath(QCoreApplication::applicationDirPath());	
 
-	QGuiApplication::setApplicationDisplayName(app.tr("qtFRED v%1").arg(FS_VERSION_FULL));
+	QGuiApplication::setApplicationDisplayName(QApplication::tr("qtFRED v%1").arg(FS_VERSION_FULL));
```

Comment:
Introducing subsystems cutscenes, mainhalls, and ranks, so that valid (available) briefing anims, mainhalls etc. become available to populate combo boxes, instead of typing them out by hand. [this commit](https://github.com/scp-fs2open/fs2open.github.com/pull/3392/commits/61fb9c5dd1fc25cfc3d98dc468d53ad1340a5e55)
`app.tr()` correctly led to clang warning about `tr` being static to QApplication, not a member of app. IIRC it also did so for adjacent subsystem lines, leading to a failed PR Check.